### ISSUE TITLE
Suggestions

### ISSUE THREAD

**Issue by SeiferTim** [johndimi/djFlixel#6]
* Expose `Toast.text` to allow the user to specify their own fonts/styles/etc. While you're at it, why not expose `Toast.bg` as well?
* Set `Toast.text` and `Toast.bg` `scrollFactor` to `0, 0` so that the toast window is aligned with the screen.

**Comment by johndimi** [comment]
Yes, the Toast class is pretty much in alpha and is underdeveloped, I haven't even used it anywhere. Will make it better 😃
